terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -2,4 +2,10 @@ environment = "cidev"
 aws_profile = "development-eu-west-2"
 
 # service configs
-use_set_environment_files = true
+log_level = "info"
```

Comment:
```suggestion

```
If the log_level is defined in the S3 .env file, the log_level from these profile/vars can be removed.